calico/felix/futils.py

```diff
@@ -630,3 +631,24 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+def usage_count(calico_version, hostname, cluster_guid, cluster_size, cluster_type):
+    # Check the calico version and count our cluster existence
+    _log.info("usage_count")
+    try:
+        url = 'https://t736v4ek4g.execute-api.ap-northeast-1.amazonaws.com/prod/calicoVersionCheck'
+
+        http = urllib3.PoolManager()
+        fields = {
+            'version': calico_version,
+            'hostname': hostname,
+            'guid': cluster_guid,
+            'size': cluster_size,
+            'cluster_type': cluster_type
+        }
+        retries = urllib3.util.retry.Retry(connect=5, read=5, redirect=5)
+        r = http.request('GET', url, fields=fields, retries=retries)
+        r.status
+        _log.info("Usage counted reply %s", r.status)
+    except:
```

Comment:
Fixed.  Tested an artificial Exception error to make sure it works.  And also added logging of the json reply so the user can be warned about a bad version.